Dockerfile

```diff
@@ -1,23 +1,28 @@
 FROM node:lts-alpine3.18 AS build-step
 ARG DYNAMIC_CONFIG=true
+ARG historyMode="history"
+ARG pathPrefix="/"
 
 WORKDIR /app
+RUN echo ${pathPrefix} > ./pathprefix
```

Comment:
I think it is possible to pass [args between buildstages](https://stackoverflow.com/a/53682110) without writing to a file

Reply:
Good pointer, did it as instructed, thanks.